src/main/kotlin/study/racingcar/domain/Winners.kt

```diff
@@ -1,14 +1,13 @@
 package study.racingcar.domain
 
 class Winners(
-    racedCars: List<Car>
-) {
-    var winners: List<Car>? = racedCars
+    racedCars: List<Car>,
+    val winners: List<Car> = racedCars
         .groupBy { it.moveCount }
-        .maxBy { it.key }?.value
-        private set
+        .maxBy { it.key }?.value!!
```

Comment:
```suggestion
        .maxBy { it.key }?.value ?: throw IllegalStateException()
```
maxBy의 entry가 null일 경우는 없어보입니다 :) 
null일 경우 exception을 던져주면 좋을 것 같아요!

Reply:
https://github.com/next-step/kotlin-racingcar/pull/227/commits/943d8d16716130cda713b4cad064583bc043ad1c
피드백 반영해보았습니다.